src/Framework/PaymentGateways/Routes/GatewayRoute.php

```diff
@@ -193,4 +193,22 @@ private function handleGatewayRouteMethod(PaymentGateway $gateway, $method, $que
             );
         }
     }
+
+    /**
+     * @param PaymentGateway $gateway
+     * @param string $method
+     * @param array $queryParams
+     *
+     * @return RedirectResponse
+     * @throws PaymentGatewayException
+     * @throws Exception
+     */
+    private function executeRouteCallback(PaymentGateway $gateway, $method, $queryParams)
+    {
+        if ($gateway->supportsSubscriptions()) {
+            return $gateway->subscriptionModule->$method($queryParams);
+        }
```

Comment:
We need to check that the method exists on the module before attempting to call it. Otherwise this will never allow routes to reach the base gateway. Please update the tests to include both.

Reply:
@JasonTheAdams I have tested to check whether subscription module routes register to the main gateway class. 
We have conditions earlier than this logic that validate the existence of the route method.

Let me know question